src/nginx.c

```diff
@@ -97,7 +99,9 @@ static int config_set (char **var, const char *value)
 
 static int config (const char *key, const char *value)
 {
-  if (strcasecmp (key, "url") == 0)
+  if (strcasecmp (key, "host") == 0)
```

Comment:
There's an opportunity for a refactoring here, but we probably want to make a minimally invasive change, so let's skip that for now.